setup.py

```diff
@@ -15,7 +15,10 @@
 # dev requirements
 tests_require = set(x.strip() for x in open('dev_requirements.txt'))
 tests_require_replacements = {
-    'https://github.com/ethereum/serpent/tarball/develop': 'ethereum-serpent>=1.8.1'}
+    # 'https://github.com/ethereum/serpent/tarball/develop': 'ethereum-serpent>=1.8.1',
+    # THIS SHOULD NOT BE MERGED
+    'https://github.com/pipermerriam/serpent/tarball/piper/add-travis-ci-testing': 'ethereum-serpent>=1.8.1',
```

Comment:
This should **NOT** be merged.  It's just here to get the tests running on python three until https://github.com/ethereum/serpent/pull/102 is merged.